components/shared/gallery/BreadcrumbsFilter.vue

```diff
@@ -36,15 +34,17 @@ const { replaceUrl } = useReplaceUrl()
 const { $i18n } = useNuxtApp()
 
 const breads = computed(() => {
-  const filters = { ...route.query }
-  delete filters.redesign
-  return filters
+  const query = { ...route.query }
+  delete query.redesign
+  delete query.search
+  const activeFilters = Object.entries(query).filter(
+    ([key, value]) => value == 'true'
+  )
+  return { ...Object.fromEntries(activeFilters), search: route.query.search }
 })
 
 const isAnyFilterActive = computed(
-  () =>
-    Boolean(breads.value?.search) ||
-    Object.values(breads.value).includes('true')
+  () => Boolean(breads.value?.search) || Object.keys(breads.value).length > 1
```

Comment:
```suggestion
  () => Boolean(breads.value?.search) || Object.keys(breads.value).length
```

Reply:
it needs to be larger then 1 because 'search' key will exist, possibly with undefined value
another option is 
``` 
() => Boolean(breads.value?.search) || Object.values(breads.value).includes('true')
```

Reply:
Aight this computed ref is a bit hard to read but I'll pretend I didn't see it